spec/spec_helper.rb

```diff
@@ -5,17 +5,25 @@
 
 CENTOS_7 = {
   platform: 'centos',
+<<<<<<< HEAD
+  version: '7.4.1708',
+=======
   version: '7.4.1708'
+>>>>>>> d31a4c49b047fc8a9a3d60b41786421899b97436
```

Comment:
Looks like you have some residual merge conflict fixes that need to be addressed.